Sources/XCTest/XCTAssert.swift

```diff
@@ -100,15 +100,15 @@ private func _XCTEvaluateAssertion(assertion: _XCTAssertion, @autoclosure messag
     }
 }
 
-/// This function emits a test failure if the general Bool expression passed
-/// to it evaluates to false.
+/// This function emits a test failure if the general `Boolean` expression passed
+/// to it evaluates to `false`.
```

Comment:
I understand your point. What I don't understand though is: if `Boolean` is not a defined type, how can it be used as one?:
`@autoclosure expression: () throws -> Boolean`

This code was taken from XCTAssert.swift